docs/milestone-1.md

```diff
@@ -0,0 +1,17 @@
+# Milestone 1
+## Context
+The milestone’s target is to be able to demonstrate the automatic provisioning
+of a Storage Tenant to a VMware Cloud Director tenant organizations using
+vCloud Director UI.
+
+## Acceptance criteria
+The acceptance criteria for the milestone are:
+-   The CI produces a docker image and stores it in the Scality registry
+-   The image can be run and contain the OSE
+-   The container can be integrated with a vault instance
+-   The integration with a vault instance is documented
+-   The container can be integrated with a vCloud Director instance
```

Comment:
Again, I would say that the service (the component, whatever fits) can be integrated with vcloud director. container is just a way we use to deploy it.

Reply:
I updated the acceptance criteria, is it better?